components/admin_console/license_settings/license_settings.test.tsx

```diff
@@ -112,24 +113,26 @@ describe('components/admin_console/license_settings/LicenseSettings', () => {
             upgradeToE0Status: jest.fn().mockImplementation(() => Promise.resolve({percentage: 0, error: null})),
         };
         const props = {...defaultProps, enterpriseReady: false, actions};
-        const wrapper = shallow(<LicenseSettings {...props}/>);
+        const wrapper = shallow<LicenseSettings>(<LicenseSettings {...props}/>);
 
         expect(actions.getLicenseConfig).toBeCalledTimes(1);
         expect(actions.upgradeToE0Status).toBeCalledTimes(1);
         actions.upgradeToE0Status = jest.fn().mockImplementation(() => Promise.resolve({percentage: 1, error: null}));
 
         const instance = wrapper.instance();
+        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+        // @ts-ignore
         expect(instance.interval).toBe(null);
         expect(wrapper.state('upgradingPercentage')).toBe(0);
 
-        await instance.handleUpgrade({preventDefault: jest.fn()});
+        await instance.handleUpgrade({preventDefault: jest.fn()} as unknown as React.MouseEvent<HTMLButtonElement>);
         expect(actions.upgradeToE0).toBeCalledTimes(1);
         expect(actions.upgradeToE0Status).toBeCalledTimes(1);
         wrapper.update();
-        jest.setTimeout(() => {
```

Comment:
I deleted the `jest.setTimeout` because it only expects a single input which is a time. Please let me know if I am missing something here